apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1/SubscriberClientImpl.SingleChannel.cs

```diff
@@ -310,7 +312,8 @@ private void StartStreamingPull()
         // Backoff delay (if present) has already been done; no need to delay here.
         private void HandleStartStreamingPullWithoutBackoff()
         {
-            _pull = _client.StreamingPull(CallSettings.FromCancellationToken(_softStopCts.Token));
+            var callSettings = CallSettings.FromHeader(XGoogRequestParamsHeaderKey, string.Format("subscription={0}", _subscriptionName));
+            _pull = _client.StreamingPull(callSettings.MergedWith(CallSettings.FromCancellationToken(_softStopCts.Token)));
```

Comment:
Instead of calling `callSettings.MergedWith(...)`, just `callSettings.WithCancellationToken(_softStopCts.Token)`.

(I'd create suggestions for these, but they'd end up with the wrong email address causing more bureaucracy...)